tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -374,7 +374,20 @@ module.exports = {
         .waitForElementNotPresent('@loadingIndicator')
         .waitForElementPresent(this.getFileRowSelectorByFileName(element))
     },
-
+    assertSharingIsDisabled: function () {
```

Comment:
This function is used to assert sharing is disabled entirely but it seems it just checks the first file in the list. I think it would be better if you pass a file name and check sharing is disabled on that file.